cloud/src/recycler/recycler.cpp

```diff
@@ -1119,6 +1200,12 @@ int InstanceRecycler::recycle_tablets(int64_t table_id, int64_t index_id, int64_
             return -1;
         }
         int64_t tablet_id = tablet_meta_pb.tablet_id();
+
+        if (!check_lazy_txn_finished(txn_kv_, instance_id_, tablet_meta_pb.tablet_id())) {
+            LOG(WARNING) << "lazy txn not finished tablet_id=" << tablet_meta_pb.tablet_id();
```

Comment:
LOG 
"lazy txn not finished, skip recycling. tablet_id= ... txn_id ..."
and add a bvar to monitor `skip_recyle_tablet_lazy_txn`

Reply:
> LOG "lazy txn not finished, skip recycling. tablet_id= ... txn_id ..." and add a bvar to monitor `skip_recyle_tablet_lazy_txn`

int the `check_lazy_txn_finished `, every abnormal branch has more detail log